src/thriveopedia/fossilisation/MuseumCard.tscn

```diff
@@ -88,6 +89,24 @@ __meta__ = {
 "_editor_description_": "PLACEHOLDER"
 }
 
+[node name="HBoxContainer" type="HBoxContainer" parent="MarginContainer/VBoxContainer/PanelContainer/MarginContainer"]
+margin_left = 2.0
+margin_top = 2.0
+margin_right = 164.0
+margin_bottom = 19.0
+alignment = 2
+
+[node name="Delete" type="TextureButton" parent="MarginContainer/VBoxContainer/PanelContainer/MarginContainer/HBoxContainer"]
```

Comment:
I'll fix this myself, but now testing I noticed that there is now mouse hover style for this. So it is confusing that an interactable X doesn't react to mouse hover like in a CustomDialog.